README.md

```diff
@@ -45,31 +44,51 @@ Radiks.js creates and manages 'signing keys' that it uses to sign all writes tha
 
-To install and run MongoDB, visit their [download page](https://www.mongodb.com/download-center/community). You can also download MongoDB using your favorite package manager. On Mac OS, [homebrew is recommended](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/#install-mongodb-community-edition-with-homebrew).
+In the future, Radiks-server will support various different databases, but right now only MongoDB is supported. 
 
-**You must use MongoDB >=3.6, because they fixed an issue with naming patterns in keys.**
+1. Install and run MongoDB 3.6 or higher.
 
-### Built-in CLI Server
+   **You must use MongoDB >=3.6, because they fixed an issue with naming patterns in keys.**
 
-The easiest way to run radiks-server is to use the pre-packaged node.js server that is included with this npm package. To use it, first install `radiks-server`:
+   If you are testing on a local workstation, you can install locally or us a `docker` image.  To install, visit their [download page](https://www.mongodb.com/download-center/community). You can also download MongoDB using your favorite package manager. On Mac OS, [homebrew is recommended](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/#install-mongodb-community-edition-with-homebrew).
 
-```bash
-npm install -g radiks-server
-# or, if you use yarn
-yarn global add radiks-server
-```
+2. On your MongoDB instance, create a database for your application data.
+  
+3. Create a username/password combination with `root` privileges on your new database.
+
+4. Install the `radiks-server` on a workstation or server.
+
+    ```bash
+    npm install -g radiks-server
+    ```
+    Or, if you prefer `yarn`:
+
+    ```bash
+    yarn global add radiks-server
+    ```
+
+5. Create an `MONGODB_URI` environment variable on the same machine where you are running the `radiks-server`.
+
+   Use the `mongodb://username:password@host:port/db_name` format for your variable. For example, to set this variable in a `bash` shell:
+
+   ```bash
+   export MONGODB_URI="mongodb://admin:mongome@157.245.167.8:27017/mycoolapp"
+   ```
+
+   The default port for Mongodb is `27017`, your instance may be configured differently.  By default, Radiks-server will use `'mongodb://localhost:27017/radiks-server'` as the `MONGODB_URI` value. This is suitable for local testing, but in production, you'll want to change the hostname and possible the database name.
+
+6. Run `radiks-server` in the command line to start a server. 
 
-After installing, you can simply run `radiks-server` in the command line to start a server. It defaults to running on port `1260`, but you can use the `PORT` environment variable to modify this.
+   The `radiks-server` defaults to running on port `1260`, but you can use the `PORT` environment variable to modify this.
 
-### Specifying the MongoDB URL
+7. Add the the radiks-server to your application's [UserSession](https://blockstack.github.io/blockstack.js/classes/appconfig.html) configuration.
```

Comment:
This is the only section that is technically incorrect. You need to pass your Radiks server URL to the Radiks `configure` method, which also accepts a `userSession` as an argument:

https://github.com/blockstack-radiks/radiks#configuration